apps/dashboard/src/app/(dashboard)/article/write-form.tsx

```diff
@@ -0,0 +1,60 @@
+import { type ArticleWrite, ArticleWriteSchema } from "@dotkomonline/types"
+import { createTextareaInput, createTextInput, useFormBuilder } from "../../form"
+
+const ARTICLE_FORM_DEFAULT_VALUES: Partial<ArticleWrite> = {}
+
+interface UseArticleWriteFormProps {
+  onSubmit(data: ArticleWrite): void
+  defaultValues?: Partial<ArticleWrite>
+  label?: string
+}
+
+export const useArticleWriteForm = ({
+  onSubmit,
+  label = "Legg inn ny artikkel",
+  defaultValues = ARTICLE_FORM_DEFAULT_VALUES,
+}: UseArticleWriteFormProps) =>
+  useFormBuilder({
+    schema: ArticleWriteSchema,
+    defaultValues,
+    onSubmit,
+    label,
+    fields: {
+      title: createTextInput({
+        label: "Tittel",
+        placeholder: "Fadderuka 2023",
+        withAsterisk: true,
+      }),
+      author: createTextInput({
+        label: "Forfattere",
+        placeholder: "Ola Nordmann, Trond-Viggo Torgersen",
```

Comment:
And then, when we have the user, why not have the ow_user id on the article as well?

Reply:
I had a bit of a misunderstanding about what was possible to do with ow_user. You can ignore the comment